### PR TITLE
{data}[intel/2019a] netcdf4-python v1.5.2 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3105.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/c0bcc3a77cc1e50e0d0e30bd2417296d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2585.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0b058c8b303c5a10a19f02fb11217f58 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2071.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9938e2c4805a6fa195f9915e21c72597 for a full test report.

File: easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.2-intel-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+homepage = 'https://unidata.github.io/netcdf4-python/'
+description = """Python/numpy interface to netCDF."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/Unidata/netcdf4-python/archive/']
+
+dependencies = [
+    ('Python', '3.7.2'),
```
Comment (Micket) [inline easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.2-intel-2019a.eb]: forgot multi_deps?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3116.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/49004df751302b86ec85d9289313830b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2700.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/eaecb0d1a2c55dbfcccb3d1b35d7f08d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/aae4ba0df82ee27c5acf140c3b369627 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/701d6a63a414617f76675ad4df03f17a for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**